src/Microsoft.AspNetCore.Server.Kestrel/Internal/Http/FrameOfT.cs

```diff
@@ -188,6 +209,10 @@ public class Frame<TContext> : Frame
                 // SetBadRequestState logs the error.
                 SetBadRequestState(ex);
             }
+            catch (IOException ex) when (ex.InnerException is UvException)
```

Comment:
This isn't for that. The try/catch above handles that and completely swallows it if it's incomplete. This replaces the "Connection processing ended abnormally" with a better one which has an eventId, connectionId, etc... Since we know a bad connection can cause SocketInput to throw and cause request processing to end abnormally, it's better to handle that explicitly. This catch will handle any connection error, not just resets.

`Log.LogWarning(0,...` is generally reserved for things we don't expect to happen under normalish circumstances.